packages/protractor/package.json

```diff
@@ -61,7 +61,10 @@
     "@serenity-js/jasmine": "^2.29.0",
     "@serenity-js/mocha": "^2.29.0",
     "protractor": "^5.0.0 || ^7.0.0",
-    "selenium-webdriver": "^3.6.0"
+    "selenium-webdriver": "^3.6.0",
+    "@types/chai": "^4.2.21",
```

Comment:
I don't think those need to be listed as `peerDependencies`, `devDependencies` should be enough